tests/containers/container_suseconnect.pm

```diff
@@ -0,0 +1,69 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: container-suseconnect test for multiple container runtimes
+#
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use version_utils qw(is_opensuse);
+use utils;
+
+my $runtime;
+
+sub run {
+    my ($self, $args) = @_;
+
+    my $runtime_name = $args->{runtime};
+
+    select_serial_terminal;
+
+    assert_script_run("curl " . data_url('containers/container-suseconnect/Dockerfile') . " -o ./Dockerfile");
+
+    $runtime = $self->containers_factory($runtime_name);
+
+    my $build_cmd = $runtime_name =~ /podman/i
+      ? "buildah bud --layers"
+      : "DOCKER_BUILDKIT=1 docker build";
+
+    my $scc_credentials_path = '/etc/zypp/credentials.d/SCCcredentials';
+    my $suseconnect_path = '/etc/SUSEConnect';
+
+    my $image_tag = "suseconnect-test-$runtime_name";
+    my $container_cmd = "$runtime_name run --rm $image_tag";
+
+    if ($runtime_name =~ /podman/i) {
+        my $buildah_installed = script_output("which buildah || echo 'not installed'");
+        if ($buildah_installed =~ /not installed/) {
+            record_info("Installing buildah");
+            zypper_call('in buildah');
+        }
+    }
```

Comment:
This is a nit-pick, but I think this can be simplified as:

```suggestion
    if ($runtime_name =~ /podman/i && script_run("command -v buildah")) {
        record_info("Installing buildah");
        zypper_call('in buildah');
    }
```

Why `command`?  Because in JeOS/MinimalVM `which` is not available by default and `command` is POSIX and provided by the shell.